### PR TITLE
Updating front matter fields and descriptions in docs.md to coordinate with metadata

### PR DESCRIPTION
**Pull request by rosemm**
Here's it rendered in lia: https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/docs_update_metdata/docs.md#7

### CODE REVIEW

**Comment by rosemm** [comment]
@drelliche I think this is ready to be merged, right? Or was there something we were waiting on?

File: docs.md [hunk 1]
```diff
@@ -255,17 +398,324 @@ Note that `@learning_objectives` is a [block macro](https://liascript.github.io/
...
+True/False based on whether achieving the module's learning objectives requires coding. This includes a user running code locally or interacting with code in the module. 
+
+Required.
+
+Must be one of `true` or `false`.
+
+### `coding_level`
+
+```
+coding_level: getting started
```
Comment (drelliche) [inline docs.md:605]:
```suggestion
coding_level: getting_started
```

File: docs.md [hunk 2]
```diff
@@ -255,17 +398,324 @@ Note that `@learning_objectives` is a [block macro](https://liascript.github.io/
...
+
+Not required.
+
+Must be one or more of the following:
+
+- r
+- python
+- bash
+- SQL
+
```
Comment (drelliche) [inline docs.md:634]:
```suggestion
- git
```

**Review by drelliche — COMMENTED** [review]
Two tiny things I found, then it looks ready to merge!

**Comment by rosemm** [comment]
Thanks so much, @drelliche! Accepted both suggestions. Good eye!

File: docs.md [hunk 3]
```diff
@@ -255,17 +398,324 @@ Note that `@learning_objectives` is a [block macro](https://liascript.github.io/
...
+- r
+- python
+- bash
+- SQL
+- git
+### `sequence_name`
+
+```
+sequence_name: bash basics
```
Comment (drelliche) [inline docs.md:638]:
```suggestion
sequence_name: bash_basics
```
Found another one!

**Review by drelliche — APPROVED** [review]
LGTM

**Pull request merged**